cli/standalone.rs

```diff
@@ -183,6 +183,7 @@ async fn run(source_code: String, metadata: Metadata) -> Result<(), AnyError> {
     ts_version: version::TYPESCRIPT.to_string(),
     no_color: !colors::use_color(),
     get_error_class_fn: Some(&crate::errors::get_error_class_name),
+    location: None,
```

Comment:
If `location` is taken from `flags.location` below why is it not passed here?

Reply:
Ah, this is from before runtime flags were passable.